lib/src/bluetooth_service.dart

```diff
@@ -122,4 +122,28 @@ class BluetoothService {
       rethrow;
     }
   }
+
+  ///
+  /// Get a list of characteristics from this service.
+  ///
+  /// [uuid] optional uuid.
+  ///
+  /// TODO: Needs Error checking
+  ///
+  ///
+  Future<List<BluetoothCharacteristic>> getCharacteristics(
+      {String? uuid}) async {
+    try {
+      final List<WebBluetoothRemoteGATTCharacteristic> characteristic =
+          await _bluetoothService.getCharacteristics(uuid);
+      List<BluetoothCharacteristic> characteristicList = [];
+      characteristic.forEach((element) {
+        characteristicList.add(BluetoothCharacteristic(element));
+      });
```

Comment:
The linter is complaining about the use of `forEach` saying that `for (final element in characteristic)` is preferred.